plenum/test/view_change_with_delays/test_delayed_instance_changes.py

```diff
@@ -23,6 +23,7 @@ def check_no_ic_msgs(node, expected_view_no, nodes):
         assert not node.view_changer.instance_changes.has_inst_chng_from(expected_view_no, n)
 
 
+@pytest.mark.skip(reason="INDY-2262. Need to select the same primaries for all nodes")
```

Comment:
Why is it passed before?

Reply:
it passed because we used select_primaries for all the batches. But it will be wrong for some cases. Now it's waiting for fix in the scope of INDY-2244 (do select primaries during view_change).